src/bin/cargo/commands/package.rs

```diff
@@ -5,6 +5,8 @@ use cargo::ops::{self, PackageOpts};
 pub fn cli() -> Command {
```

Comment:
Something I think forgot to ask earlier is whether we should insta- stabilize this or make it unstable with a testing period before stabilizing.

Thoughts?

Reply:
Sure, that makes sense, especially since there could be some unforeseen interactions with publishing workspaces. I've made it unstable, and also made the new `--registry` and `--index` flags unstable for `package`.

Quick question about the tests: do we want to duplicate some or all of them so that we can see the difference with/without the unstable flag?

Reply:
Since this has behavior under stable Cargo, probably

Reply:
Ok, I didn't duplicate all of the tests because some of them differ in ways that are only interesting when `-Zpackage-workspace` is set, but I did a few.